src/main/java/com/sarahhand/fractals/juliaconstants/ChangeJuliaConstants.java

```diff
@@ -0,0 +1,74 @@
+package com.sarahhand.fractals.juliaconstants;
+
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
+import com.sarahhand.fractals.view.FractalsUI;
+
+public class ChangeJuliaConstants {
+
+	private Map<String, Float> juliaConstants;
+
+	private JFrame frame;
+
+	private JPanel constantPanelsPanel;
+
+	private JButton done;
+
+	private volatile boolean finished = false;
+
+	public Map<String, Float> getJuliaConstants() {
+		return juliaConstants;
+	}
+
+	public boolean isFinished() {
+		return finished;
+	}
+
+	private List<ConstantsPanel> constantPanels;
+
+	public ChangeJuliaConstants(FractalsUI ui) {
+		frame = new JFrame("Julia Constants");
+		frame.setLayout(new BorderLayout());
+		frame.setSize(500, 500);
```

Comment:
Can you make the frame size smaller since the constants being set takes up only a small section of the window?